### PR TITLE
Update compliance certification pages

### PR DESCRIPTION
**Pull request by jxainguyen**
Update certification pages:
* Remove CSF from HITRUST pages and nav-bar
* Update not-in-scope tables for HIPAA, ISO, PCI, TISAX, SOC2
* Update HIPAA caution callout verbiage

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx [hunk 1]
```diff
@@ -10,14 +10,10 @@ freshnessValidatedDate: never
...
 
-<Callout variant="important">
-  To use New Relic's offering in compliance with US HIPAA regulations, contact your account representative regarding a Business Associate Addendum (BAA).
-</Callout>
-
 ## Applicable document by service [#applicable-services]
 
 <Callout variant="caution">
-  Not all [New Relic Observability Platform](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one/) services are in compliance with the HIPAA-compliant program. For more information, please see [services included](/docs/security/security-privacy/compliance/hitrust/).
+Not all [New Relic Observability Platform](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one/) services are in compliance with this program. For non-compliant services, please see the section of [services not in scope](#not-scope)
```
Comment (vpayyapilly) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx:16]: ...section on...

**Review by vpayyapilly — APPROVED** [review]

File: src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx (hunk 1 above)
Comment (sujitnewrelic) [inline src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/hipaa.mdx:16]: period

**Comment by sujitnewrelic** [comment]
Add "Page resources" to the "browser.yml" file:

Go to src/nav/browser.yml and append the following after the "Marks and measures" title and path.

      - title: Marks resources
        path: /docs/browser/new-relic-browser/browser-pro-features/page-resources

**Review by sujitnewrelic — APPROVED** [review]
lgtm

**Pull request merged**